programs/locked-voter/src/instructions/partial_merge_partial_unstaking.rs

```diff
@@ -0,0 +1,70 @@
+use crate::*;
+
+/// Accounts for [voter::merge_partial_unstaking].
+#[derive(Accounts)]
+pub struct PartialMergePartialUnstaking<'info> {
```

Comment:
Why not use the same context with `MergePartialUnstaking`, caused all accounts are the same